collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/ResetProjectTest.kt

```diff
@@ -127,4 +137,17 @@ class ResetApplicationTest {
             .assertText(R.string.use_device_language)
             .assertTextDoesNotExist("español")
     }
+
+    @Test
+    fun when_rotateScreen_should_resetDialogNotDisappear() {
+        rule.startAtFirstLaunch()
+            .clickTryCollect()
+            .openProjectSettingsDialog()
+            .clickSettings()
+            .clickProjectManagement()
+            .clickOnResetApplication()
```

Comment:
Please rename this method to `resetProject` to make it consistent with the class name that you renamed.